src/context/WalletProvider/MobileWallet/components/MobileSuccess.tsx

```diff
@@ -63,7 +49,16 @@ export const MobileSuccess = ({ location }: MobileSetupProps) => {
       // Make sure the component is completely unmounted before we revoke the mnemonic
       setTimeout(() => vault?.revoke(), 500)
     }
-  }, [appDispatch, dispatch, getAdapter, localWallet, setIsSuccessful, setWelcomeModal, vault])
```

Comment:
issue (blocking): revert me, and potentially follow-up with a separate diff.

rationale: as ugly as the current reactivity may look, it's like so for historical reasons.
Given the huge risks this PR introduces (and regressions found after testing), can we keep the diff to the bare minimum to fix the issue at hand, so that:
1. we derisk wallet issues in this PR
2. we can dedicate a follow-up PR and its testing to having full deps
3. if reviewers are happy with follow-up but both ops and eng. ever miss a but that's spotted some time later, we can bisect it and revert the reactivity changes commit only, not this whole diff including reactivity changes + the fix from this PR

Reply:
Not valid anymore as discussed we prefer the quickfix regarding the complexity of `WalletProvider`, changed in [5eacb01](https://github.com/shapeshift/web/pull/7280/commits/5eacb015dbea0fa87977f66ebe16bb50b6338894)